atoma-proxy/src/server/handlers/chat_completions.rs

```diff
@@ -169,11 +169,25 @@ pub async fn chat_completions_create(
             .and_then(serde_json::Value::as_bool)
             .unwrap_or_default();
 
+        let wallet_address = state
+            .sui
+            .write()
+            .await
+            .get_wallet_address()
+            .unwrap_or_default()
+            .to_string();
+
         match handle_chat_completions_request(&state, &metadata, headers, payload, is_streaming)
             .await
         {
             Ok(response) => {
-                TOTAL_COMPLETED_REQUESTS.add(1, &[KeyValue::new("model", metadata.model_name)]);
+                TOTAL_COMPLETED_REQUESTS.add(
+                    1,
+                    &[
+                        KeyValue::new("model", metadata.model_name),
+                        KeyValue::new("wallet_address", wallet_address),
```

Comment:
This doesn't make that much sense, since the proxy wallet is static (never changes), so we shouldn't filter by its value. On the other hand, we should filter by model, as before

Reply:
Correct, my bad I conflated the proxy wallet with the user's wallet, which in fact is our `user_id`